extensions/Khronos/KHR_lights/README.md

```diff
@@ -0,0 +1,144 @@
+# KHR\_lights
+
+## Contributors
+
+* Norbert Nopper, UX3D, <mailto:nopper@ux3d.io>
+* Thomas Kress, UX3D, <mailto:kress@ux3d.io>
+* Mike Bond, Adobe, <mailto:mbond@adobe.com>
+
+## Status
+
+Draft (not ratified yet)
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines a set of lights for use with glTF 2.0. 
+
+Many 3D tools and engines support built-in implementations of light types. Using this extension, tools can export and engines can import these lights. 
+
+This extension defines four light types: `ambient`, `directional`, `point` and `spot`.
+
+## Lights
+
+Lights define light sources within a scene.
+
+`ambient` lights are not transformable and, thus, can only be defined on the scene. All other lights are contained in nodes and inherit the transform of that node.
+
+A conforming implementation of this extension must be able to load light data defined in the asset and has to render the asset using those lights. 
+
+### Defining Lights
+
+Lights are defined within an dictionary property in the glTF manifest file, by adding an `extensions` property to the top-level glTF 2.0 object and defining a `KHR_lights` property with a `lights` array inside it.
+
+Each light defines a mandatory `type` property that designates the type of light (`ambient`, `directional`, `point` or `spot`). The following example defines a white-colored directional light.
+
+```javascript
+"extensions": {
+    "KHR_lights" : {
+        "lights": [
+            {
+                "color": [
+                    1.0,
+                    1.0,
+                    1.0
+                ],
+                "type": "directional"
+            }
+        ]
+    }
+}
+```
+
+### Adding Light Instances to Nodes
+
+Nodes can have lights attached to them, just like any other objects that have a transform. Only lights of types `directional`, `point`, and `spot` have position and/or orientation so only these light types are allowed. These lights are attached to a node by defining the `extensions.KHR_lights` property and, within that, an index into the `lights` array using the `light` property.
+
+```javascript
+"nodes" : [
+    {
+        "extensions" : {
+            "KHR_lights" : {
+                "light" : 0
+            }
+        }
+    }            
+]
+```
+
+For light types that have a position (`point` and `spot` lights), the light's position is defined as the node's world location.
+For light types that have a direction (`directional` and `spot` lights), the light's direction is defined as the 3-vector `(0.0, 0.0, 1.0)` and the rotation of the node orients the light accordingly.
```

Comment:
Does this mean the direction in a world space will be "light node's world rotation matrix * Vector3(0.0, 0.0, 1.0)"?